pkg/daemon/ceph/osd/daemon.go

```diff
@@ -319,9 +318,26 @@ func getAvailableDevices(context *clusterd.Context, desiredDevices []DesiredDevi
 		//
 		// So earlier lsblk extracted the '/dev' path, hence the device.Name property
 		// device.Name can be 'xvdca', later this is formated to '/dev/xvdca'
-		isAvailable, rejectedReason, err := sys.CheckIfDeviceAvailable(context.Executor, device.RealPath, pvcBacked)
-		if err != nil {
-			return nil, errors.Wrapf(err, "failed to get device %q info", device.Name)
+		var err error
+		var isAvailable bool
+		rejectedReason := ""
+		if agent.pvcBacked {
+			block := fmt.Sprintf("/mnt/%s", agent.nodeName)
```

Comment:
What about keeping `device.RealPath`? (use `/dev/` then).

Reply:
I was doing the same thing as the other place where `getCephVolumeRawOSDs()` is called [here](https://github.com/rook/rook/blob/db12fc86fc0a3ee0b8580d4e362301963aa3dec9/pkg/daemon/ceph/osd/volume.go#L119). Should that be changed as well?